src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write code here
+document.querySelectorAll('li').forEach((li) => {
+  const span = document.createElement('span');
+
+  span.textContent = li.firstChild.textContent;
+  li.replaceChild(span, li.firstChild);
+
+  span.addEventListener('click', (e) => {
+    const children = li.querySelector('ul');
+
+    if (children) {
+      children.style.display =
+        children.style.display === 'none'
+          ? (children.style.display = 'block')
+          : (children.style.display = 'none');
```

Comment:
The ternary operator is used to toggle the `children.style.display` property, but the assignment within the ternary operator is redundant. You can simplify this by directly assigning the result of the ternary operation to `children.style.display` like this: `children.style.display = children.style.display === 'none' ? 'block' : 'none';`